.github/workflows/ci.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
     name: Build and test azure-init
     runs-on: ubuntu-latest
     steps:
+      - name: Install libudev
+        run: |
+          sudo apt install -y libudev-dev
```

Comment:
should always do an apt update unless we know that it has been done in an earlier step

Reply:
Done, thanks